frontend/apps/pwa/src/reducers/Recorder/recorder.adapter.ts

```diff
@@ -0,0 +1,10 @@
+import { createEntityAdapter } from '@reduxjs/toolkit';
+
+import { FormValue } from './types';
+
+export const adapter = createEntityAdapter<FormValue>({
+  // Assume IDs are stored in a field other than `book.id`
```

Comment:
Are these comments from an example? Should probably make them more relevant or remove them

Reply:
I didn't add them myself, I'll what I can do ^^